src/fake-timers-src.js

```diff
@@ -895,9 +895,17 @@ function withGlobal(_global) {
                     ? nativeHandler(timerId)
                     : undefined;
             }
+
+            // Include the stacktrace, excluding the 'error' line
+            const stackTrace = new Error().stack
+                .split("\n")
+                .slice(1)
+                .join("\n");
+
```

Comment:
Unlike `warnOnce`, this relatively heavy operation will be calculated on each hit, but given that using native timers should be a relatively infrequent occurence, so it's fine.